pipelines/_ingredients/ApPipeWithFakes.yaml

```diff
@@ -119,11 +124,21 @@ tasks:
       connections.fakesType: parameters.fakesType
       connections.science: fakes_initial_pvi
 subsets:
+  apPipeSingleFrame:
+    subset:
+      - isr
+      - calibrateImage
+      - initialPviCore
+    description: >-
+      The prompt ApPipe tasks that make up single-frame processing. Not to be confused with the
+      SingleFrame.yaml pipeline, which is designed to be run instead of ApPipeWithFakes.yaml and
+      includes some preload tasks and some non-ApPipe tasks.
```

Comment:
Suggested reword: "Not to be confused with `SingleFrame.yaml`, which includes `apPipeSingleFrame`, and is designed as an alternative to `ApPipeWithFakes.yaml` in the situation where templates for difference imaging are not available.

Reply:
I'm not sure I agree:
* Saying that `SingleFrame.yaml` includes `apPipeSingleFrame`, while technically true, makes them sound related, which is exactly the confusion I wanted to avoid.
* I don't want to say that `SingleFrame.yaml` is for "the situation where templates are not available", and objected to that wording when it was put in the file. Describing components as being for some assumed context (in this case, for a package that `ap_pipe` isn't supposed to depend on) leads to nonmodularity and future bugs. As far as I'm concerned, `SingleFrame` is a standalone pipeline that's not a subset of `ApPipe`, no more, no less.

Reply:
OK fair, I appreciate you want to avoid conflating "what it is" and "what we often use it for" in docs like this.

How about, "Not to be confused with the `SingleFrame.yaml` pipeline, which does more than just ApPipe single frame processing, and is designed as an alternative to `ApPipeWithFakes.yaml`." ?